packages/interactive-messages/src/http-handler.ts

```diff
@@ -141,3 +139,38 @@ export function createHTTPHandler(adapter) {
       });
   };
 }
+
+export type HTTPHandler = (req: IncomingMessage & { body?: string, rawBody?: Buffer }, res: ServerResponse) => void;
```

Comment:
is there a meaningful difference between this type and `RequestListener` (provided by `@types/node`)?

Reply:
I didn't know this type existed 😮 Neat!